pkg/image/controller/trigger/image_trigger_controller_test.go

```diff
@@ -773,6 +798,28 @@ func scenario_1_buildConfig_imageSource() *buildv1.BuildConfig {
 	}
 }
 
+func scenario_1_buildConfig_imageSource_pre48Cluster() *buildv1.BuildConfig {
```

Comment:
Can you please add a godoc here explaining the scenario? As I understand it the scenario is:

1. A BuildConfig was created with an ImageChangeTrigger policy before OCP 4.8
2. The BuildConfig was triggered at least once before upgrading to OCP 4.8

Reply:
that is correct

will add godoc momentarily

Reply:
one minor clarification ... the last triggered image ID is up to date 

`From:                 &corev1.ObjectReference{Kind: "ImageStreamTag", Name: "stream:2", Namespace: "other"},`

and 

`LastTriggeredImageID: "image/result:2"`

that will also be captured in the upcoming godoc

Reply:
will also try to better capture this in the method name

Reply:
sorry I have that backwards ... it will be sorted out in the update I push